docs/rum_collection/crash_reporting.md

```diff
@@ -92,7 +92,7 @@ Depending on your setup, you may need to download dSYM files from App Store Conn
 
 ### Upload your dYSM file
 
-By uploading your dYSM file to Datadog, you gain access to the file path, line number, and code snippet of each frame in an error's related stack trace.
+By uploading your dYSM file to Datadog, you gain access to the file path and line number of each frame in an error's related stack trace.
```

Comment:
Actually I'm a bit confused since `dSYM` seems misspelled as `dYSM` here. Is this file the source if I want to update https://docs.datadoghq.com/real_user_monitoring/ios/crash_reporting/?tab=cocoapods#upload-your-dsym-file ?
cc @priyanshi-gupta ?